cfme/tests/control/test_actions.py

```diff
@@ -597,22 +596,16 @@ def finalize():
 
 
 @pytest.mark.provider([VMwareProvider, RHEVMProvider], scope="module")
-@pytest.mark.meta(
-    blockers=[
-        BZ(
-            1549529,
-            forced_streams=["5.10", "upstream"],
```

Comment:
I can see that this  BZ(1549529) is CLOSED CURRENTRELEASE but It will be better, if we add this BZ(1549529) under *Bugzilla* data.

Reply:
@ganeshhubale I didn't put it there because it is the same BZ as 1745065, the original bug was closed but it's still an issue.

Reply:
Yes agree But I would put this BZ for more info around this test case. Although this is somehow related to this test case.

Reply:
Okay, I can put it in there :+1: